backend/src/main/java/io/metersphere/track/service/TestPlanReportService.java

```diff
@@ -532,8 +528,8 @@ public TestPlanReport finishedTestPlanReport(String testPlanReportId, String sta
                             }
                         }
                         content.setPlanScenarioReportStruct(JSONObject.toJSONString(scenarioCases));
-                    }catch (Exception e){
-                        LogUtil.error("update test plan api error! "+e.getMessage());
+                    } catch (Exception e) {
+                        LogUtil.error("update test plan api error! " + e.getMessage());
```

Comment:
maybe this is better：LogUtil.error("update test plan api error! " ,  e.getMessage());